### DISCUSSION TITLE
How to change output?

### DISCUSSION THREAD

**Discussion by narFnarF** [File-New-Project/EarTrumpet#1327] [Q&A] [answered]
Hi
My computer has 2 outputs: built-in speakers and headphones when I plug them in. Can I change the output using EarTrumper like I can using the default Windows widget?

Note: I'm talking about changing the default output, so that all my apps output in the speakers or the headphones.

**Comment by creative-resort** [comment, marked as answer]
On Windows 10/11:
Simply right-click the eartrumpet taskbar icon and you'll be presented with the built-in selector for the default device.

In addition to that, on Win 11 you can (left-)click the native speaker icon and then click the sound-device icon to the right of the volume slider, to get a default device selector, as well.
